### ISSUE TITLE
0.6 RC2: cache/local growing infinitely at fast pace

### ISSUE THREAD

**Issue by rednil** [RetroShare/RetroShare#156]
Revision 8551: /cache/local rapidly fills up with files like "fc-own-1445595798.rsfc". In my case, each of them is 4MB, and a new one is generated every 15 minutes (folder settings). This adds up to almost 2GB PER DAY.

**Comment by realityfabric** [comment]
Is this a problem in versions earlier or later than the latest release on SourceForge? My installation on Windows 10 of RC2 from SourceForge doesn't have this problem.

**Comment by rednil** [comment]
As i wrote, it was revision 8551, the one delivered from csoler/stable. Following on your question, I tried to install the one from ppa:retroshare/stable, but apt complains that it cannot be authenticated, so I won't install it. Regarding the issue, I couldn't observe it at a friends installation (he has fewer files than I do), and I can't observe it right now (I changed the interval to 400min instead of 15min, and currently my cache/local dir contains only 6 days worth of cache files = 222MB)

**Comment by realityfabric** [comment]
As far as I can tell GitHub doesn't allow for searching by revision number, and I'm not sure how to compare between svn and git

**Comment by AsamK** [comment]
rev 8551 is 89187bd453dbf0112dcee05cb0e71bd18875b6bd tagged v0.6.0-RC2-SVN in git

**Comment by rednil** [comment]
Ok, so I am on RC2. I just reduced the interval again to 15min (the default), and I can still see the problem: Files are accumulating in /cache/local, I am already at 331MB (up from 222MB 2 hours ago)

**Comment by csoler** [comment]
I'm surprised by the "I cannot authenticate package" in the new ppa. Have you add it using "apt-add-repository" or did you just download the .deb? This type of issue should be easy to fix, if there is.

**Comment by rednil** [comment]
Me too ;-) I added it via sudo add-apt-repository ppa:retroshare/stable, and apt-key list gives me

```
/etc/apt/trusted.gpg.d/retroshare-stable.gpg
--------------------------------------------
pub   4096R/144729B5 2015-08-02
uid                  Launchpad PPA for RetroShare Team
```

**Comment by csoler** [comment]
I bumped into this once already: the PGP key that is published on the ppa page (0932399B) is the key with which we sign packages to be uploaded, so that launchpad can check that it's us who upload.

The PGP key of the debian archive of the ppa is a different key. It's used by you to check that the binaries made by launchpad are correct. I do not believe I have any way to update/change this key that is generated by launchpad.

Now, if "apt-get install" complains about signatures, that's a problem, and it shouldn't! When I use the unstable ppa it does not. I havn't tried the other ppa recently though, but it's pretty much the same thing.
